From c7f2ad6fd03907003b294bd3f87ec7be0b6a8c50 Mon Sep 17 00:00:00 2001
From: Ethan Lee <flibitijibibo@gmail.com>
Date: Apr 19 2024 14:39:45 +0000
Subject: AMD HDR


---

diff --git a/kernel.spec b/kernel.spec
index bb7d523..ceabc52 100644
--- a/kernel.spec
+++ b/kernel.spec
@@ -163,16 +163,18 @@ Summary: The Linux kernel
 %define specrpmversion 6.8.7
 %define specversion 6.8.7
 %define patchversion 6.8
-%define pkgrelease 300
+%define pkgrelease 301
 %define kversion 6
 %define tarfile_release 6.8.7
 # This is needed to do merge window version magic
 %define patchlevel 8
 # This allows pkg_release to have configurable %%{?dist} tag
-%define specrelease 300%{?buildid}%{?dist}
+%define specrelease 301%{?buildid}%{?dist}
 # This defines the kabi tarball version
 %define kabiversion 6.8.7
 
+%define kcflags -DAMD_PRIVATE_COLOR
+
 # If this variable is set to 1, a bpf selftests build failure will cause a
 # fatal kernel package build error
 %define selftests_must_build 0

